sections/leaderboard/Leaderboard/Leaderboard.tsx

```diff
@@ -85,9 +88,9 @@ const Leaderboard: FC<LeaderboardProps> = ({ compact }: LeaderboardProps) => {
 				pnl: (pnlMap[stat.account]?.pnl ?? wei(0)).toNumber(),
 			}))
 			.filter((i: { trader: string }) =>
-				searchTerm?.length ? i.trader.toLowerCase().includes(searchTerm) : true
+				searchTerm?.length ? i.trader.toLowerCase().includes(addressOrENS) : true
```

Comment:
Why not just do an "or" with two checks here? If the search term looks like the address OR the ENS name, return that record.

Will make more sense with the changes in [this PR](https://github.com/Kwenta/kwenta/pull/947)

Should be able to do something like:
```
searchTerm?.length ?
    i.trader.toLowerCase().includes(searchTerm) || 
    i.traderEns.toLowerCase().includes(searchTerm) 
    : true
```